### PR TITLE
feat(components): [check-tag] add `disabled` prop

### PR DESCRIPTION
**Pull request by HaceraI**
Please make sure these boxes are checked before submitting your PR, thank you!

- [✔] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [✔] Make sure you are merging your commits to `dev` branch.
- [✔] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/tag.md [hunk 1]
```diff
@@ -105,10 +105,11 @@ tag/checkable
...
 ### CheckTag Attributes
 
-| Name                      | Description      | Type                                                               | Default |
-| ------------------------- | ---------------- | ------------------------------------------------------------------ | ------- |
-| checked / v-model:checked | is checked       | ^[boolean]                                                         | false   |
-| type ^(2.5.4)             | type of CheckTag | ^[enum]`'primary' \| 'success' \| 'info' \| 'warning' \| 'danger'` | primary |
+| Name                      | Description                       | Type                                                               | Default |
+|---------------------------|-----------------------------------| ------------------------------------------------------------------ | ------- |
+| checked / v-model:checked | is checked                        | ^[boolean]                                                         | false   |
+| disabled                  | whether the check-tag is disabled | ^[boolean]                                                         | false   |
```
Comment (btea) [inline docs/en-US/component/tag.md:111]:
```suggestion
| disabled    ^(2.8.1)              | whether the check-tag is disabled | ^[boolean]                                                         | false   |
```

File: docs/en-US/component/tag.md [hunk 2]
```diff
@@ -105,10 +105,11 @@ tag/checkable
...
 ### CheckTag Attributes
 
-| Name                      | Description      | Type                                                               | Default |
-| ------------------------- | ---------------- | ------------------------------------------------------------------ | ------- |
-| checked / v-model:checked | is checked       | ^[boolean]                                                         | false   |
-| type ^(2.5.4)             | type of CheckTag | ^[enum]`'primary' \| 'success' \| 'info' \| 'warning' \| 'danger'` | primary |
+| Name                      | Description                       | Type                                                               | Default |
+|---------------------------|-----------------------------------| ------------------------------------------------------------------ | ------- |
+| checked / v-model:checked | is checked                        | ^[boolean]                                                         | false   |
+| disabled    ^(2.8.1)              | whether the check-tag is disabled | ^[boolean]                                                         | false   |
```
Comment (btea) [inline docs/en-US/component/tag.md:111]: Now the table format may look messy. You can run `pnpm format` to adjust it.

Comment (HaceraI) [reply docs/en-US/component/tag.md]: okay, I have completed it.

**Review by btea — APPROVED** [review]
LGTM

File: docs/en-US/component/tag.md [hunk 3]
```diff
@@ -105,10 +105,11 @@ tag/checkable
...
 ### CheckTag Attributes
 
-| Name                      | Description      | Type                                                               | Default |
-| ------------------------- | ---------------- | ------------------------------------------------------------------ | ------- |
-| checked / v-model:checked | is checked       | ^[boolean]                                                         | false   |
-| type ^(2.5.4)             | type of CheckTag | ^[enum]`'primary' \| 'success' \| 'info' \| 'warning' \| 'danger'` | primary |
+| Name                      | Description                       | Type                                                               | Default |
+| ------------------------- | --------------------------------- | ------------------------------------------------------------------ | ------- |
+| checked / v-model:checked | is checked                        | ^[boolean]                                                         | false   |
+| disabled ^(2.8.1)         | whether the check-tag is disabled | ^[boolean]                                                         | false   |
```
Comment (tolking) [inline docs/en-US/component/tag.md:111]:
```suggestion
| disabled ^(2.8.2)         | whether the check-tag is disabled | ^[boolean]                                                         | false   |
```

**Review by tolking — APPROVED** [review]

**Pull request merged**